hack/aws-image-job.yaml

```diff
@@ -21,7 +21,7 @@ spec:
 
       initContainers:
       - name: payload
-        image: registry.redhat.io/openshift-sandboxed-containers/osc-podvm-payload-rhel9:1.5.2
+        image: registry.redhat.io/openshift-sandboxed-containers/osc-podvm-payload-rhel9:1.6.0
```

Comment:
These are no longer needed for 1.6.0. The aws-image-job.yaml and azure-image-job.yaml can be removed

Reply:
Additionally the tag in these files needs to be changed to 1.6.0
https://github.com/openshift/sandboxed-containers-operator/blob/devel/config/peerpods/podvm/osc-podvm-create-job.yaml#L18
https://github.com/openshift/sandboxed-containers-operator/blob/devel/config/peerpods/podvm/osc-podvm-create-job.yaml#L32
https://github.com/openshift/sandboxed-containers-operator/blob/devel/config/peerpods/podvm/osc-podvm-delete-job.yaml#L22
https://github.com/openshift/sandboxed-containers-operator/blob/devel/config/peerpods/podvm/osc-podvm-gallery-delete-job.yaml#L15